### PR TITLE
develop initial release

### PR DESCRIPTION
**Pull request by aptalca**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  make && \
+  make install && \
+  mkdir -p /usr/share/licenses/unrar && \
+  mv license.txt /usr/share/licenses/unrar/ && \
+  echo "**** install pyload ****" && \
+  if [ -z ${PYLOAD_COMMIT+x} ]; then \
+    PYLOAD_COMMIT=$(curl -sX GET "https://api.github.com/repos/pyload/pyload/branches/develop" \
+	    | awk '/sha/{print $4;exit}' FS='[""]'); \
+  fi && \
+  pip3 install -U pip setuptools wheel && \
```
Comment (TheSpad) [inline Dockerfile:48]: Do we want to do a `  python3 -m pip install --upgrade pip` as a matter of course with pip builds?

Comment (aptalca) [reply Dockerfile:48]: this line does the same thing

Comment (TheSpad) [reply Dockerfile:48]: Sorry, not wering my glasses and missed the second pip in the line.

**Review by TheSpad — APPROVED** [review]
LGTM

**Pull request merged**